NOTICES.md

```diff
@@ -895,14 +1444,14 @@ https://github.com/oldrev/Sandwych.QuickGraph
     (E) The software is licensed "as-is." You bear the risk of using it. The contributors give no express warranties, guarantees or conditions. You may have additional consumer rights under your local laws which this license cannot change. To the extent permitted under your local laws, the contributors exclude the implied warranties of merchantability, fitness for a particular purpose and non-infringement.
 
 ## Selenium  
```

Comment:
removed selenium

Reply:
we are using Selenium.Support  & Selenium.WebDriver ,so I updated it